README.md

```diff
@@ -171,3 +176,23 @@ $client->push($message);
 ### GCM
 
   - [Everything you should know about GCM payload](https://developers.google.com/cloud-messaging/http-server-ref)
+
+
+
+## Tests
+
+We want this library to be fully covered by unit tests, so if you contribute to this project, be aware that your PR build will fail if the code coverage drops below 100%.
+
+### Running tests with HTML code coverage
+
+```bash
+# install dependencies
+composer install
+mkdir -p wallet
+
+# if xdebug extension is enabled on PHP CLI
+vendor/bin/phpunit --coverage-html wallet/coverage
```

Comment:
I think it would add a lot of useless complexity (most of folks won't need a VM to contribute to this project), so that's the best idea I came with. Besides the `wallet` folder is already gitignored :)